packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1449,7 +1444,7 @@ export class ContainerRuntime
 		existing: boolean,
 		blobManagerSnapshot: IBlobManagerLoadInfo,
 		private readonly _storage: IDocumentStorageService,
-		private readonly createIdCompressor: () => Promise<IIdCompressor & IIdCompressorCore>,
+		private readonly createIdCompressorFn: () => IIdCompressor & IIdCompressorCore,
```

Comment:
This is ctor argument/class member that holds the 'createIdCompressorFn' callback we saw above.

I've renamed it 'createIdCompressor' -> 'createIdCompressorFn' to avoid confusion with the 'createIdCompressor' function imported from the 'id-compressor' package.

Per the above, the callback also now returns the IIdCompressor instance synchronously (no promise.)